ansible/playbooks/templates/custom-coredns-helmchart.yaml.j2

```diff
@@ -17,7 +17,7 @@ spec:
     image:
       repository: registry.k8s.io/coredns/coredns
       tag: v1.10.1
-    replicaCount: 3
+    replicaCount: 1
```

Comment:
Is there a way to dynamically adjust this, based on master or worker count? Obviously if you're running a single node cluster, there's no point running more than 1, but if you're running something like a triple master, it might be worth scaling this up.

Reply:
Not easily, since this would need to be a template and then copied from the template folder to the final directory.

Reply:
Can't we do an shell task and use sed to replace this with a variable?

Reply:
We could but I think it's ok to just leave it as is like we do for ingress-nginx, k8s_gateway, echo-server etc... If there's an easier way I am all ears but This is one of those things people will want to customize on their own.